.github/workflows/build.yaml

```diff
@@ -277,21 +284,23 @@ jobs:
           key: ${{ runner.os }}-binaries-tf-${{ github.sha }}
 
       - name: Install dependencies
+        env:
+          LIBTENSORFLOW_VERSION: "2.12.1"
```

Comment:
Do we still use tensorflow? Don't want to get rid of it everywhere?

Reply:
Yeah, if we can tear it out without breaking builds then let's do it. I started trying in https://github.com/livepeer/go-livepeer/pull/2836 but haven't had time to finish it

Reply:
oh! that requires quite a lot of changes in go code... which i'd rather not touch in this small-ish PR. would rather bump 2836 with changes from this PR and keep tensorflow removal over in that alone.